### PR TITLE
Update egg TS3-Manager

### PR DESCRIPTION
**Pull request by gOOvER**
# Description

- update egg to PTDL_v2
- update images
- update installscript

## Checklist for all submissions

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you tested and reviewed your changes with confidence that everything works?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

* [x] You verify that the start command applied does not use a shell script
  * [ ] If some script is needed then it is part of a current yolk or a PR to add one
* [x] The egg was exported from the panel

### CODE REVIEW

File: voice_servers/ts3_manager/egg-t-s3-manager.json [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
...te\r\napt install -y git unzip jq wget tar curl\r\n\r\n## env\r\nexport HOME=\/mnt\/server\r\ncd $HOME\r\n\r\n## get release info and download links\r\nLATEST_RELEASE=$(curl -L -s -H 'Accept: application\/json' https:\/\/github.com\/${GITHUB_PACKAGE}\/releases\/latest)\r\nLATEST_VERSION=$(echo $LATEST_RELEASE | sed -e 's\/.*\"tag_name\":\"\\([^\"]*\\)\".*\/\\1\/')\r\n\r\nif [ -z \"${VERSION}\" ] || [ \"${VERSION}\" == \"latest\" ]; then\r\n    DOWNLOAD_LINK=\"https:\/\/github.com\/${GITHUB_PACKAGE}\/releases\/download\/$LATEST_VERSION\/$MATCH-$LATEST_VERSION\"\r\nelse \r\n    DOWNLOAD_LINK=\"https:\/\/github.com\/${GITHUB_PACKAGE}\/releases\/download\/v$VERSION\/$MATCH-$VERSION-v$VERSION\"\r\nfi\r\n\r\necho $DOWNLOAD_LINK\r\nwget $DOWNLOAD_LINK\r\n\r\nif [ -z \"${VERSION}\" ] || [ \"${VERSION}\" == \"latest\" ]; then\r\n    mv -f $MATCH-$LATEST_VERSION $HOME\/start_ts3-manager\r\n    chmod +x start_ts3-manager\r\nelse\r\n    mv -f $MATCH-v$VERSION $HOME\/start_ts3-manager\r\n    chmod start_ts3-manager\r\nfi\r\n\r\necho \"-------------------------------------------------------\"\r\necho \"Installation completed\"\r\necho \"-------------------------------------------------------\"",
```
Comment (QuintenQVD0) [inline voice_servers/ts3_manager/egg-t-s3-manager.json:25]: This should not start with
`# Valheim Installation Script`

**Review by QuintenQVD0 — CHANGES_REQUESTED** [review]

File: voice_servers/ts3_manager/egg-t-s3-manager.json (hunk 1 above)
Comment (gOOvER) [reply voice_servers/ts3_manager/egg-t-s3-manager.json:25]: arg, i didnt get a notification :/

Comment (gOOvER) [reply voice_servers/ts3_manager/egg-t-s3-manager.json]: fixed

**Review by QuintenQVD0 — APPROVED** [review]
LGTM

**Pull request merged**